net/sslsplit/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2016-2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sslsplit
+PKG_VERSION:=0.5.2
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE LICENSE.contrib LICENSE.third
+
+PKG_SOURCE:=sslsplit.tar.gz
```

Comment:
Use release tarball instead

Reply:
Do you mean I should rename sslsplit.tar.gz to sslsplit-0.5.2.tar.gz?

Reply:
https://mirror.roe.ch/rel/sslsplit/sslsplit-0.5.2.tar.bz2